pages/dkp/kommander/2.3/applications/appdeployments/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: layout.pug
+navigationTitle: AppDeployments
+title: AppDeployments
+excerpt: Use AppDeployments to deploy, and customize platform, DKP catalog, and custom applications in your environment
+menuWeight: 10
+beta: false
+enterprise: false
+---
+
+An `AppDeployment` is a [CustomResource][CRD] created by D2iQ with the purpose of deploying applications (platform, DKP catalog and custom applications) in management, managed clusters, or both. Customers of both Essential and Enterprise products use `AppDeployments`, regardless of their setup (networked, air-gapped, etc.), and their infrastructure provider.
+
+When installing DKP, an `AppDeployment` resource is created for each enabled **platform application**. This `AppDeployment` resource references a `ClusterApp`, which then references the repository that contains a concrete declarative and preconfigured setup of an application, usually in the form of a [`HelmRelease`][helm]. `ClusterApps` are cluster-scoped so that these platform applications are deployable to all workspaces or projects.
+
+In the case of **DKP catalog** and **custom applications**, the `AppDeployment` references an `App` instead of a `ClusterApp`, which also references the repository containing the installation and deployment information. `Apps` are namespace-scoped and are meant to only be deployable to the workspace or project scope in which they have been created.
```

Comment:
```suggestion
In the case of **DKP catalog** and **custom applications**, the `AppDeployment` references an `App` instead of a `ClusterApp`, which also references the repository containing the installation and deployment information. `Apps` are namespace-scoped and are meant to only be deployable to the workspace or project in which they have been created.
```